### PR TITLE
Feature sub header menu

### PR DESCRIPTION
**Pull request by dhaval2989**
submenu feature

### CODE REVIEW

File: src/app/layout/nv-sub-menu/nv-sub-menu.component.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+@Component({
+  selector: 'app-nv-sub-menu',
+  templateUrl: './nv-sub-menu.component.html',
+  styleUrls: ['./nv-sub-menu.component.scss']
+})
+export class NvSubMenuComponent implements OnInit {
+  menuItems = ['menu item 1', 'menu item 2'];
+  siteItems = ['site item 1', 'site item 2'];
+  actionItems = ['action 1', 'action 2'];
```
Comment (Sina-pz) [inline src/app/layout/nv-sub-menu/nv-sub-menu.component.ts:13]: I think these three variable are no longer needed

**Review by Sina-pz — APPROVED** [review]
Please address the one comment I mentioned
I already approved it.
Good Job

File: src/app/layout/nv-sub-menu/nv-sub-menu.component.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  /**
+   * DropDowns event handlers
+   */
+  onMenuItemSelect(item: string): void {
+    console.log('selectedMenuItem >>>> ' + +item);
```
Comment (veeranjaneyuludavuluri) [inline src/app/layout/nv-sub-menu/nv-sub-menu.component.ts:39]: remove console logs and commented code if there is any

File: src/app/layout/nv-sub-menu/nv-sub-menu.component.ts (hunk 1 above)
Comment (dhaval2989) [reply src/app/layout/nv-sub-menu/nv-sub-menu.component.ts:13]: Remove menu items

File: src/styles.scss [hunk 3]
```diff
@@ -3,7 +3,7 @@
 @import "./../src/app/reusable/theme/base.scss";
 
 html, body { height: 100%; }
-body { margin: 0; font-family: "Lato", Roboto, "Helvetica Neue", sans-serif;  overflow-y: hidden;}
+body { margin: 0; font-family: "Lato", Roboto, "Helvetica Neue", sans-serif;  overflow-y: auto;}
```
[the remark this answers is not in the document]
Comment (dhaval2989) [reply src/styles.scss]: resolved.

File: src/app/layout/nv-sub-menu/nv-sub-menu.component.ts (hunk 2 above)
Comment (dhaval2989) [reply src/app/layout/nv-sub-menu/nv-sub-menu.component.ts]: resolved

**Review by veeranjaneyuludavuluri — APPROVED** [review]
good to go

**Pull request merged**